exotations/exotica_core_task_maps/src/task_map_py.cpp

```diff
@@ -68,15 +96,15 @@ PYBIND11_MODULE(exotica_core_task_maps_py, module)
         .def_readwrite("joint_ref", &Identity::joint_ref_);
 
     py::class_<IMesh, std::shared_ptr<IMesh>, TaskMap>(module, "IMesh")
-        .def_property("W", &IMesh::get_weights, &IMesh::set_weights)
-        .def("set_weight", &IMesh::set_weight)
+        .def_property("W", &IMesh::GetWeights, &IMesh::SetWeights)
+        .def("SetWeight", &IMesh::SetWeight)
```

Comment:
Nope, this is incorrect please revert. Python is methods with underscore (I had this fixed in the earlier one)

Reply:
Unresolved